Cakes-Co/style.css

```diff
@@ -1 +1,344 @@
-/*  Add your styling here */
+:root {
+  --background: #de9bab;
+  --brand: #36000b;
+  --layout-color: #d1758b;
+  --brand-accent: #263600;
+  --transition-effect: ease-in-out 0.4s;
+}
+
+body {
+  background-color: var(--background);
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--brand);
+}
+
+a {
+  text-decoration: none;
+}
+
+.cupcake-logo {
+  border-radius: 100%;
+}
+
+header p {
+  color: var(--brand);
+}
+
+main div:nth-child(1) {
+  background-image: url("https://images.unsplash.com/photo-1588195538326-c5b1e9f80a1b?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=3150&q=80");
```

Comment:
Nowadays, people don't use background image so much. They are more likely to use the grid strategy to make full screen images. 

Look into this and tell me why.